test/visibility/submodule-in-file/errors/WrongModule2Wrong.good

```diff
@@ -0,0 +1 @@
+error: Cannot include module from module with different file name
```

Comment:
Seeing it in context, I think this error message could be clearer (assuming we keep it).  Maybe "Cannot include modules from a module whose name doesn't match its filename?

Reply:
Done